torchvision/datasets/caltech.py

```diff
@@ -18,10 +19,16 @@ class Caltech101(VisionDataset):
     Args:
         root (string): Root directory of dataset where directory
             ``caltech101`` exists or will be saved to if download is set to True.
-        target_type (string or list, optional): Type of target to use, ``category`` or
-            ``annotation``. Can also be a list to output a tuple with all specified
-            target types.  ``category`` represents the target class, and
-            ``annotation`` is a list of points from a hand-generated outline.
+        target_type (string or list, optional): Type of target to use, ``category``, ``annotation``, ``box_coord``, or
+            ``obj_contours``. Can also be a list to output a tuple with all specified target types. The targets
+            represent:
+
+                - ``category`` (int): Target class
+                - ``annotation`` (dict): Raw annotation including the bounding box in Y1Y2X1X2 format and the contour
+                  vertices in relative coordinates to the bounding box.
+                - ``box_coord`` (np.ndarray, shape=(1, 4), dtype=int): Bounding box in XYXY format
+                - ``obj_contours`` (np.ndarray, shape=(N, 2) dtype=float): Contour vertices in XY format
```

Comment:
Do we really need to support individual box_coord and obj_contours? Can't we let users use the keys that they want in the dict?

Regardless, we really want the format to be consistent. Let's just use XYXY everywhere instead of leaving the annotation "raw".